selfdrive/locationd/locationd.cc

```diff
@@ -529,6 +535,9 @@ void Localizer::time_check(double current_time) {
   if (std::isnan(this->last_reset_time)) {
     this->last_reset_time = current_time;
   }
+  if (std::isnan(this->start_time)) {
+    this->start_time = current_time;
+  }
```

Comment:
@HaraldSchafer do you want to measure from first gpsLocation message, or from first of any of the messages?